### DISCUSSION TITLE
Overlay over background

### DISCUSSION THREAD

**Discussion by pierre-oord** [orestbida/cookieconsent#118] [Ideas]
I would like my website's visitors to make a choice, instead of ignoring the cookie consent dialog. It would therefore be great if we could put a dark overlay over the website, while the dialog is shown. After choosing an option, the overlay could be removed.

One question that comes to mind, is if this would influence Google's page rank. I am no expert in this area.

**Comment by orestbida** [comment]
Hi @pierre-oord,

have you tried enabling the [`force_consent`](https://github.com/orestbida/cookieconsent#all-available-options) option?

If you're worried about your page rank, you can entirely disable the plugin whenever a google bot/crawler is detected, via the `hide_from_bots` option.

**Reply by pierre-oord** [reply]
Oh that's working perfectly! "force consent" sounded like "the customer is required to accept all cookies", but after I've tested it now, it actually still allow people to reject them all (not giving consent?). Might be good to add this to the documentation as well! Anyhow thanks, it's working great and I really appreciate your help!

**Reply by ac-zebrakm** [reply]
The thing you were afraid of is called "Cookie wall" and is prohibited in EU (at least European Privacy Data Board say so). "Force_consent" parameter is more like "Force decission".
